keras_contrib/preprocessing/image_3d.py

```diff
@@ -0,0 +1,633 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+import warnings
+from keras import backend as K
+from keras.preprocessing.image import ImageDataGenerator, Iterator
+import scipy.ndimage as ndi
+
+
+def make_rotation_matrix(rotation_range_value, choice_number):
+    """
+    Make roation Matrix (x, y, z axis)
+    Reference:
+        https://www.tutorialspoint.com/computer_graphics/
+               3d_transformation.htm
+    :param rotation_range_value: setting roation degree
+    :param choice_number: which axis choose
+    :return: rotation matrix
+    """
+    if rotation_range_value != 0.0:
+        theta = np.pi / 180 * rotation_range_value
+    else:
+        theta = 0
+
+    if choice_number == 1:
+        rotation_matrix_x = np.array([[1, 0, 0, 0],
+                                      [0, np.cos(theta), -np.sin(theta), 0],
+                                      [0, np.sin(theta), np.cos(theta), 0],
+                                      [0, 0, 0, 1]])
+        return rotation_matrix_x
+    elif choice_number == 2:
+        rotation_matrix_y = np.array([[np.cos(theta), 0, np.sin(theta), 0],
+                                      [0, 1, 0, 0],
+                                      [-np.sin(theta), 0, np.cos(theta), 0],
+                                      [0, 0, 0, 1]])
+        return rotation_matrix_y
+    elif choice_number == 3:
+        rotation_matrix_z = np.array([[np.cos(theta), -np.sin(theta), 0, 0],
+                                      [np.sin(theta), -np.cos(theta), 0, 0],
+                                      [0, 0, 1, 0],
+                                      [0, 0, 0, 1]])
+        return rotation_matrix_z
+
+
+def make_shift_matrix(x,
```

Comment:
how could one know what this does fromt the comments?

Reply:
I'll fix comment